packages/sqlduck/README.md

```
@@ -128,75 +128,79 @@ const queryResult = await dbDuckDbMemoryConn.query<{
-    1.04x faster than rowToColumnsChunk with transformer with chunkSize 2048 (count: 100000)
+    1.21x faster than rowToColumnsChunk with transformer with chunkSize 2048 (count: 100000)
     1.28x faster than mapFakeRowStream with chunkSize 2048 (count: 100000)
-
-  getTableCreateFromZod - bench/table-create.bench.ts > Bench getTableCreateFromZod
-
 ```
 
-### Bun 1.3.13
+### Bun 1.4.0-dev
 
 ```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)